src/App.css

```diff
@@ -40,3 +48,342 @@
+  }
+  
+  .navList li {
+    margin-right: 20px; /* Add space between each nav item */
+  }
+  
+  .todoForm{
+    display: flex;
+    gap: 10px;
+    margin-bottom: 20px;
+    align-items: center;
+    font-size: 2rem;
+  }
+  
+  .title{
+    margin-right:100px;
+  }
+  
+  .removeButton{
+    background-color: rgb(221, 241, 221);
+    color: #000;
+    height: 25px;
+    font-size: 18px;
+    align-items: baseline;
+    padding: 5px;
+    margin: 5px;
+  }
+  
+  .addButton{
+    background-color: rgb(221, 241, 221);
+    color: #000;
+    height: 25px;
+    font-weight: bold;
+    font-size: 15px;
+    align-items: baseline;
+    padding: 5px;
+    margin: 5px;
+  }
+}
+
+
+/* Styles for small desktops */
+@media (min-width: 1025px) and (max-width: 1200px){
+  .navbar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 1.5rem;
+    font-weight: 200;
+    color: honeydew;
+    padding: 1em;
+  }
+
+  .logo {
+    height: 3em;
+    padding: 1.5em;
+
+  }
+  
+  .navList {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    margin: 0;
+  }
+  
+  .navList li {
+    margin-right: 20px; /* Add space between each nav item */
+  }
+  
+  .todoForm{
+    display: flex;
+    gap: 10px;
+    margin-bottom: 20px;
+    align-items: center;
+    font-size: 2rem;
+  }
+  
+  .title{
+    margin-right:100px;
+  }
+  
+  .removeButton{
+    background-color: rgb(221, 241, 221);
+    color: #000;
+    height: 25px;
+    font-size: 18px;
+    align-items: baseline;
+    padding: 5px;
+    margin: 5px;
+  }
+  
+  .addButton{
+    background-color: rgb(221, 241, 221);
+    color: #000;
+    height: 25px;
+    font-weight: bold;
+    font-size: 15px;
+    align-items: baseline;
+    padding: 5px;
+    margin: 5px;
+  }
+}
+
+
+  /* Styles for large screens */
+
+@media (min-width: 1201px) {
+  .navbar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 1.5rem;
+    font-weight: 200;
+    color: honeydew;
+    padding: 1em;
+  }
+
+  .logo {
+    height: 3em;
+    padding: 1.5em;
+
+  }
+  
+  .navList {
+    list-style: none;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    margin: 0;
+  }
+  
+  .navList li {
+    margin-right: 20px; /* Add space between each nav item */
+  }
+  
+  .todoForm{
+    display: flex;
+    gap: 10px;
+    margin-bottom: 20px;
+    align-items: center;
+    font-size: 2rem;
+  }
+  
+  .title{
+    margin-right:100px;
+  }
+  
+  .removeButton{
+    background-color: rgb(221, 241, 221);
+    color: #000;
+    height: 25px;
+    font-size: 18px;
+    align-items: baseline;
+    padding: 5px;
+    margin: 5px;
+  }
+  
+  .addButton{
+    background-color: rgb(221, 241, 221);
+    color: #000;
+    height: 25px;
+    font-weight: bold;
+    font-size: 15px;
+    align-items: baseline;
+    padding: 5px;
+    margin: 5px;
+  }
```

Comment:
You have a ton of styles in the stylesheet, some of which are redundant. I'd recommend considering a mobile-first styling approach here instead which would likely allow you to reduce much of this code. 

TLDR: build for the smallest screen size first and then scale media queries out in terms of width from there. You'll find this to be simpler and much less verbose.

https://www.freecodecamp.org/news/taking-the-right-approach-to-responsive-web-design/